counting_sort/CountingSort.java

```diff
@@ -0,0 +1,38 @@
+import java.util.Random;
+ class CountingSort {
+	
+	public static void main(String[] args) {
+		
+		int A[]=new int[10000];
+        int k=0;
+        Random rand=new Random();
+        for(int i=0;i<A.length;i++) {
+            A[i]=rand.nextInt(100);
+            if(k<A[i])
+            	k=A[i];
+            }
+        
+        A=cSort(A,k);
+        for(int i=0;i<A.length;i++)
+        	System.out.print(A[i] + " ");
+	}
+
+	public static int[] cSort(int[] a,int k) {
```

Comment:
Rename method to filename to best extent possible as mentioned in contributing.md
You can use `countingSort()`